deep_quoridor/src/random_agents.py

```diff
@@ -0,0 +1,13 @@
+from arena import Agent
```

Comment:
nit: rename this file `random_agent.py` (singular) for consistency with `simple_agent`.
nit follow-up: move the agents to an `agents` folder, so we can keep adding there. And drop the _agent from the name.